src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  background: #e5e5e5;
+  margin: 0;
+  min-width: 375px;
+  max-width: 1200px;
+}
+
+.header {
+  box-sizing: border-box;
+  justify-content: space-between;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  font-size: 0;
+  padding-top: 6px;
+  padding-bottom: 6px;
+}
+
+.nav {
+  position: relative;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  cursor: pointer;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.nav__item {
+  list-style: none;
+  text-decoration: none;
+  text-align: center;
+  margin-left: 20px;
```

Comment:
Try to remove margin-left for first nav__item 
![image](https://user-images.githubusercontent.com/54936044/228374498-9dbb403c-75bb-400e-b180-3f33dfbe6c3e.png)